urdf2webots/parserURDF.py

```diff
@@ -28,9 +28,23 @@
 
 # to pass from external
 robotName = ''
-extensionList = ['.3ds', '.blend', '.bvh', '.dae', '.fbx', '.stl', '.obj', '.x3d']
+extensionListSingleColor = ['.stl', '.obj']
```

Comment:
I would rename this variable to make it clearer:
```suggestion
extensionListSingleAppearance = ['.stl', '.obj']
```